docs/devGuide/maintainerGuide.md

```diff
@@ -51,6 +51,8 @@ Please follow the [commit message convention](https://oss-generic.github.io/proc
 
 3. Ensure that a version milestone is tagged to the PR. **Reason:** We may have missed it during the "Approval" stage, so please add the version milestone if it is missing, so that the drafting of the release notes during the release process will be easier.
 
+**Note**: Before confirming the merging do ensure that no other PRs are merged to master **after you drafted the merging commit's title and message**. Otherwise GitHub may glitch out and merge your PR without squashing. (Reference: [MarkBind#1160](https://github.com/MarkBind/markbind/pull/1160))
```

Comment:
I think this should be under the Actual Merging section (maybe somewhere near L22?)